CHANGELOG.md

```diff
@@ -1,5 +1,27 @@
 # Azure Kubernetes Service Changelog
 
+## Release 2022-03-03
+
+This release is rolling out to all regions - estimated time for completed roll out is 2022-03-16 for public cloud and 2022-03-19 for sovereign clouds.
```

Comment:
Shouldn't the roll out be 2022-03-14 for public and 22-03-21 for sov? Did we change the release cadence?

Reply:
When I asked for the last release I was told 13 days and 16 days from the release date - please let me know if we were wrong!

Reply:
@phealy Who told you that? I might have been wrong with my dates 😅

Reply:
I will honor the rule of 13 days and 16 days from 03-03 which makes it 03-16 and 03-19 ...